src/x509_str.c

```diff
@@ -36,9 +36,20 @@
 
 #ifndef NO_CERTS
 
-/*******************************************************************************
+#ifdef OPENSSL_EXTRA
+static int X509StoreGetIssuerEx(WOLFSSL_X509 **issuer,
+                            WOLFSSL_STACK *certs, WOLFSSL_X509 *x);
+static int X509StoreAddCa(WOLFSSL_X509_STORE* store,
+                                          WOLFSSL_X509* x509, int type);
+#endif
+
+#ifndef WOLFSSL_X509_STORE_DEFAULT_MAX_DEPTH
+#define WOLFSSL_X509_STORE_DEFAULT_MAX_DEPTH 100
```

Comment:
How was the value of 100 choose? Perhaps a code comment?

Reply:
Chosen because openssl uses that as its default max depth. I will leave a comment.